leases/schema.py

```diff
@@ -65,12 +72,10 @@ class Input:
     berth_lease = graphene.Field(BerthLeaseNode)
 
     @classmethod
-    @login_required
-    @superuser_required
+    @view_permission_required(BerthLease, BerthApplication, Berth, Boat)
+    @add_permission_required(BerthLease)
```

Comment:
also interesting question: do we need to specify `view_permission_required(BerthLease)` if it is included in `add_permission_required(BerthLease)` 🤔

Reply:
I was thinking about whether to include it or not because, at least logically, the `add` would mean that you can `view` it as well.

Technically, in practice, you could have `add` perms _without_ `view` perms, so I settled for explicitly checking for both. BUT if we agree that on our logic `add`, `change` or `delete` _require_ to have `view`, then we can remove it.

Thoughts?

Reply:
> if we agree that on our logic add, change or delete require to have view, then we can remove it.

I think the code will be cleaner if we do accept this. 👍

Reply:
Alrighto! Let's go on with this then :)